explorer/test/Test/Pos/Explorer/Web/ClientTypesSpec.hs

```diff
@@ -103,9 +106,79 @@ unitTests = do
 -- Quickcheck tests
 ----------------------------------------------------------------------------
 
+-- type AddressHash   = AbstractHash Blake2b_224
+-- type Hash          = AbstractHash Blake2b_256
+-- type TxId          = Hash Tx = AbstractHash Blake2b_256 Tx
+-- type StakeholderId = AddressHash PublicKey = AbstractHash Blake2b_224 PublicKey
```

Comment:
Let's remove these ^ commented type information. Our editors can do this work for us :)